mysite/__init__.py

```diff
@@ -0,0 +1 @@
+"""Configuration files for site."""
```

Comment:
Not files: python operates such things as modules and packages. Also config files are normally some structured text files, which *don't contain any code*.